### ISSUE TITLE
MacOS downloadable link is broken

### ISSUE THREAD

**Issue by matrunchyk** [heatcrypto/heatwallet#7]
Both Client and Wallet links are broken.

Also is there a documentation/tutorial how to start?

**Comment by resultingforce** [comment]
Would you please point the broken links on the website?
Yes, the Mac OS Client app links is broken on the Readme. Will be fixed.
Thanks.

To setup the heatledger server go to https://github.com/heatcrypto/heatledger/releases
There the version is slightly outdated. The new version (with new private assets feature) is preparing.
Short guide https://github.com/heatcrypto/heatledger
If you need to know the key features and advantages of the heatledger server you can it find on the sites or requested it.

**Comment by matrunchyk** [comment]
@resultingforce thank you for your response and fixing the link!
On the main website it's also fixed: https://www.heatledger.com/heat-crypto-wallet-decentralized-exchange-dex/ (Mac OS X button).

How to configure the Wallet in order to use my server?

I had to use Java1.8 (not in the app, as a standalone server I downloaded), and in order to run the server against Java1.8 I used some Docker image:
`docker run -it --name heatledger --mount type=bind,source="$(pwd)"/,target=/app -p 7744:7744 lwieske/java-8 /app/bin/heatledger`
However, a built-in server into Heatwallet app doesn't work because of Java version mismatch.

That's what I have on my system (Mac):
```
openjdk 13.0.1 2019-10-15
OpenJDK Runtime Environment (build 13.0.1+9)
OpenJDK 64-Bit Server VM (build 13.0.1+9, mixed mode, sharing)
```

Thanks for the instructions, could you also provide more details about all options on `.properties` file?
In particular, what is a use case (ideally example) for microservices, websockets, signalling server (WebRTC)?

Thank you!!!

**Comment by matrunchyk** [comment]
Few more questions:
Which API call is used in order to generate a pubkey/secret/signature?
I noticed Node SDK on your GitHub account, do you also have PHP SDK? Is Node SDK up-to-date?

**Comment by resultingforce** [comment]
Hi Serhii,
Heat Wallet 3.1.0 was tested for Java 1.8, Java 9, Java 12 on the Windows 10, Ubuntu. Not sure that this version was tested for all java versions on Mac OS. Compatibility with java versions >1.8 (9 - 12) has implemented since 3.1.0

>  could you also provide more details about all options on .properties file?

there are a lot of notes for properties in the file conf/heat-default.properties

> what is a use case (ideally example) for microservices, websockets, signalling server (WebRTC)?

microservices see https://github.com/heatcrypto/heat-microservice

websocket is used for fast transactions sending from client to blockchain node (heatledger server) using heat-sdk. Also there is the planned feature to implement server-to-server (peer-to-peer) data exchanging using the websockets.

There is the Signaling server (webRTC) embedded  in the heatledger server, in result the blockchain node can provide establishing direct secure messaging between client apps (heatwallet or heat-ui web app). For example, there is some blockchain node with enabled signaling component, some users config their heatwallets to use this node, then the users can connect each to each via WebRTC and chatting on the page Messages. Shortly, blockchain network supplyes WebRTC signaling using existing crypto features (signaling and messages are encrypted/decrypted using account's public/private keys).

> Restarting doesn't help though (shows the same message)

need more info, logs

> Which API call is used in order to generate a pubkey/secret/signature?

the such things are generated on the client app. Heat SDK is useful for this https://github.com/heatcrypto/heat-sdk

> I noticed Node SDK on your GitHub account, do you also have PHP SDK? Is Node SDK up-to-date?

no PHP SDK. @matrunchyk please point the web link to the Node SDK?

**Comment by resultingforce** [comment]
@matrunchyk , the REST API of the heatledger server  https://heatwallet.com/api/#/

**Comment by matrunchyk** [comment]
@resultingforce Thank you for all replies! I will continue experimenting tomorrow with generating pubkeys/secrets. My goal is to generate selling/buying transaction messages and integrate the API into a PHP website I'm working on.

> no PHP SDK. @matrunchyk please point the web link to the Node SDK?
You have provided it: https://github.com/heatcrypto/heat-sdk
OK, probably I will need to use API directly with php-curl or maybe create a proxy based on NodeJs SDK.

Do you have a possibility to encrypt messages out of the box and encode them by public keys so the peers can decode them?

**Comment by resultingforce** [comment]
Yes, you can send heat transaction with encrypted text/data. Only the transaction recipient (account) can decrypt the message. Example https://github.com/heatcrypto/heat-sdk/blob/1aa3808479d489c8222f08a1a9e78fdca5511a3f/test/crypto.test.ts#L172

Also you can encrypt text without needing transaction, directly using https://github.com/heatcrypto/heat-sdk/blob/1164c873bcfc4b7fc41d1036c86088b65a4ba089/src/crypto.ts#L285

**Comment by resultingforce** [comment]
> My goal is to generate selling/buying transaction messages and integrate the API into a PHP website I'm working on.
@matrunchyk you want to use the heatwallet exchange for  selling/buying transactions (orders)?

**Comment by matrunchyk** [comment]
Hello! Which are the valid paths to the Wallet & Client for macOS?

This ones seems broken:
https://github.com/heatcrypto/heatwallet/releases/download/v3.1.0/Heatwallet_MacOS_3.1.0.zip
https://github.com/heatcrypto/heat-ui/releases/download/v3.1.0/Heatclient_MacOS_3.1.0.zip
(from README on this repo)

**Comment by resultingforce** [comment]
Hello, @matrunchyk . There is no the actual version for Mac at now. We will issue new version 4.x soon, likely the app will be for macOS also.
